src/compiler/diagnosticMessages.json

```diff
@@ -1405,7 +1405,7 @@
         "category": "Error",
         "code": 2474
     },
-    "'const' enums can only be used in property or index access expressions or the right hand side of an import declaration or export assignment.": {
+    "'const' enum can only be used in property or index access expressions, the right hand side of an import declaration or export assignment, a typeof query.": {

```

Comment:
make this

`"A 'const' enum can only be used in a property or index access expression, the right hand side of an import declaration or export assignment, or a type query."`

@JsonFreeman can potentially help with this.